src/views/Timeline.vue

```diff
@@ -2,6 +2,22 @@
 div
   h2 Timeline
 
+  div.d-flex.justify-content-between.align-items-end
+    table
+      tr
+        th.pr-2
+          label Host filter:
+        td
+            select(v-model="filter_hostname")
+              option(value='*') *
```

Comment:
Re my other comment: You might have to write `:value='null'` here, to read it as a value instead of the `"null"` string.